src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -45,6 +52,12 @@ export class RundownTimelineService implements RundownService {
     await this.rundownRepository.saveRundown(rundown)
   }
 
+  private assertNoRundownIsActive(activeRundown: BasicRundown | undefined): void {
```

Comment:
This does not assert whether there is a rundown active. It asserts if a basic rundown is given to it.